spec/league_stats_spec.rb

```diff
@@ -0,0 +1,91 @@
+require_relative 'spec_helper'
+require './lib/game'
+require './lib/team'
+require './lib/game_teams'
+require './lib/league_stats'
+
+describe LeagueStats do
+  game_path = './data/games.csv'
+  team_path = './data/teams.csv'
+  game_teams_path = './data/game_teams.csv'
+
+  data = {
+    games: game_path,
+    teams: team_path,
+    game_teams: game_teams_path
+  }
+  let(:league_stats) {league_stats = LeagueStats.new(data)}
```

Comment:
Why did you use let vs before(:each) at the top of league_stats?

Reply:
I used the let for the beginning of the spec here and changed it in the team spec as well so that it could match across more of our tests.